.github/workflows/pre-commit.yml

```diff
@@ -50,16 +53,23 @@ jobs:
           # It doesn't make sense to lint deleted files.
           # Therefore we specify we are only interested in added or modified files.
           filters: |
-            all:
+            any:
               - added|modified: '**/*'
 
       - uses: getsentry/action-setup-venv@9e3bbae3836b1b6f129955bf55a19e1d99a61c67 # v1.0.5
+        if: steps.changes.outputs.any == 'true'
         with:
           python-version: 3.8.18
           cache-dependency-path: |
             requirements-dev.txt
             requirements-dev-frozen.txt
-          install-cmd: python3 -m tools.hack_pip && pip install -r requirements-dev.txt -c requirements-dev-frozen.txt
+          install-cmd: |
+            set -ex
+            python3 -m tools.hack_pip
+            pip install \
+              -r requirements-dev.txt \
+              -c requirements-dev-frozen.txt \
+            ;
```

Comment:
I know you want to make things better but please do not change from what we know already works -- it makes things inconsistent without (much or any) benefit

Reply:
you've marked as resolved but not fixed this

Reply:
Should it go in the other direction? E.g., instead of making sentry like getsentry, make getsentry more like this?

Reply:
actually even worse -- this breaks caching because this command is used as part of the key (and therefore cannot contain newlines)